docs/utilities/parameters.md

```diff
@@ -159,260 +175,240 @@ SSM Parameter Store | `SSMProvider.getMultiple(String)`
 Secrets Manager | `SecretsProvider.get(String)` `SecretsProvider.get(String, Class)`   | `secretsmanager:GetSecretValue`
 DynamoDB | `DynamoDBProvider.get(String)` `DynamoDBProvider.getMultiple(string)` | `dynamodb:GetItem` `dynamoDB:Query`
 
-## SSM Parameter Store
+## Retrieving Parameters
```

Comment:
regarding the permissions above, I think you don't have everything (AppConfig, and maybe others? https://docs.powertools.aws.dev/lambda/python/latest/utilities/parameters/#iam-permissions)

Reply:
Added missing and made more pythonic